src/components/Navigation/Pages/PageBrowse/PageBrowse.js

```diff
@@ -71,7 +71,7 @@ class PageBrowse extends SystemManagedList {
     const target = event.target
     const activeProvider = target.name
     this.setState({ ...this.state, activeProvider })
-    this.props.dispatch(uiBrowseGet(this.props.token, activeProvider))
+    this.props.dispatch(uiBrowseGet(this.props.token, { type: activeProvider }))
```

Comment:
In the schema i've found a "provider" properties, which i guess is supposed to be used when querying for a specific provider, but here the property passed to the query is "type".

Maybe i'm missing something but i can't understand why there is such kind difference of usage.

Reply:
this aligns with the EntitySpec

see https://github.com/clearlydefined/website/blob/master/src/utils/entitySpec.js#L78

`npm/npmjs/@types/lodash/1.0.0`
`<type>/<provider>/<namespace>/<name>/<revision>`